server/models/Comment.js

```diff
@@ -0,0 +1,44 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const ReplySchema = new Schema({
+  reply: {
+    type: String,
+    required: true,
+  },
+  user: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  status: {
+    type: String,
+    required: true,
+    default: 'active',
+  },
+});
```

Comment:
add timestamps setting to replies as well.

Reply:
Done